contributors/devel/development.md

```diff
@@ -159,17 +159,14 @@ brew install coreutils ed findutils gawk gnu-sed gnu-tar grep make jq
 ```
 
 You will want to include this block or something similar at the end of
-your `.bashrc` or shell init script:
+your `.zshrc` or shell init script:
 
 ```sh
-GNUBINS="$(find `brew --prefix`/opt -type d -follow -name gnubin -print)"
-
+GNUBINS=($(find `brew --prefix`/opt -type d -follow -name gnubin -print))
 for bindir in ${GNUBINS[@]}
 do
-  export PATH=$bindir:$PATH
+  export PATH="$bindir:$PATH"
 done
-
-export PATH
 ```
 
 This ensures that the GNU tools are found first in your path. Note
```

Comment:
I don't think we've actually depended on this for a long time, see e.g. https://github.com/kubernetes/kubernetes/blob/88dfcb225d41326113990e87b11137641c121a32/hack/lib/util.sh#L729

I think we should keep the suggestion to `brew install` and drop the rest of this heading.

Reply:
Yeah, it seems a bit out of date.
Especially if these docs are referencing `bash`, as the switch to `zsh` took place with macOS Catalina in 2019.

What, you think we should remove the whole section about adding to your path and just keep the `brew install` piece at the top?

Reply:
Well again, we don't expect people to use these tools interactively, but we do have a lot of bash scripts that use them.

Reply:
Yeah, I think we just need the brew instal section.

This whole doc probably needs a review 😅

Reply:
I'll update 👍